### ISSUE TITLE
Unable to find step definitions in ruby project

### ISSUE THREAD

**Issue by serh11d** [alexkrechik/VSCucumberAutoComplete#381]
Example: https://github.com/serh11d/ruby_cucumber

```
{
	"ruby.useLanguageServer": false,
	"ruby.pathToBundler": "bundle",
	"cucumberautocomplete.steps": [
		"features/step_definitions/*_steps.rb"
	],
	"cucumberautocomplete.strictGherkinCompletion": true,
	"cucumberautocomplete.syncfeatures": "features/*.feature",
	"cucumberautocomplete.pages": {
		"_": "lib/page_objects/*.rb"
	},
	"cucumberautocomplete.gherkinDefinitionPart": "@(Given|And|When|Then|Step)\\(u"
}
```
Tried w/ and w/o "cucumberautocomplete.gherkinDefinitionPart"

Feature: https://github.com/serh11d/ruby_cucumber/blob/master/features/login.feature
Step: https://github.com/serh11d/ruby_cucumber/blob/master/features/step_definitions/login_steps.rb

**Comment by alexkrechik** [comment]
I see some problems with your configuration:
1. `cucumberautocomplete.gherkinDefinitionPart` value is wrong due to it is differs from the `And(/^I login as "([^"]*)" user$/) do |user_type|` definition type. `And` is a standard keyword so this option could be removed.
2. If `"cucumberautocomplete.strictGherkinCompletion": true,` is using, then steps, that were defined with 'And' function, will not be suggested at all, so I'm proposing to also remove this option.
3. Also, not sure that `cucumberautocomplete.pages` works as you expect.

So, should work properly with `steps ` and `syncfeatures ` cucumberautocomplete options only.

**Comment by serh11d** [comment]
Changed config to
```
{
	"cucumberautocomplete.steps": [
		"features/step_definitions/*_steps.rb"
	],
	"cucumberautocomplete.syncfeatures": "features/*.feature"
}
```
Still does not work.

**Comment by alexkrechik** [comment]
Did you try to restart IDE with a new setup? I've tried your repo and everything is looking good for me:
<img width="1489" alt="Screenshot 2021-03-02 at 18 34 57" src="https://user-images.githubusercontent.com/5772406/109681443-23b79500-7b86-11eb-9eb7-11814d40d368.png">
Thousand of users using the same step definitions format with this extension, so there is no but in it.

**Comment by serh11d** [comment]
I restarted it many times.
The only difference I see is Gemfile isn't shown with proper icon and there is no run button. Looks like it's related to my local environment. But how can I to figure out what is missed for this extension?
![rubycumber](https://user-images.githubusercontent.com/77780411/109829712-76588600-7c46-11eb-9b2b-b2da50f61816.png)

**Comment by alexkrechik** [comment]
Do you have any global extension settings installed? Anyway, please try to use the 'When' gherkin keyword and the 'When' step definition function.

**Comment by serh11d** [comment]
I do.
Ruby
https://marketplace.visualstudio.com/items?itemName=rebornix.Ruby
and
VSCode Ruby
https://marketplace.visualstudio.com/items?itemName=wingrunr21.vscode-ruby

Another thing I'm on Windows, but the most part of Ruby developers are on MacOS/Linux.

**Comment by alexkrechik** [comment]
I meant some vscucumberautocomplete global settings like `strictGherkinCompletion` and `gherkinDefinitionPart`. There are an ability to setup them for the whole app and in this case they will not work with your setup.

**Comment by serh11d** [comment]
You are right! But I don't remember that I set it before on user level (I did only for workspace). Might it set by default after extension install?

**Comment by mberrueta1** [comment]
in my case I needed to add `"cucumberautocomplete.stepRegExSymbol": "/"` because my steps are like 

`When(/^I have configured...` 

it works now
